Makefile

```diff
@@ -107,8 +107,7 @@ target/bundle/bin:
 # Create HTML docs
 
 target/bundle/doc:
-	clojure -M:doc -m lrsql.render-doc doc target/bundle/doc
-	cp -r doc/images/ target/bundle/doc/images/
```

Comment:
Don't we still need to copy the images over? Or does the new lib do that for us?

Reply:
Ok downloaded the artifact they do get copied over.